spk/dotnet-9-runtime/Makefile

```diff
@@ -0,0 +1,40 @@
+SPK_NAME = dotnet-9-runtime
+SPK_VERS = 9.0.0
+SPK_REV = 1
+SPK_ICON = src/dotnet.png
+
+OPTIONAL_DEPENDS = cross/libstdc++
+DEPENDS = cross/libicu cross/dotnet-9-runtime
+
+REQUIRED_MIN_DSM = 7.0
+
+# Arch exclusions for dotnet
+DOTNET_CORE_ARCHS = 1
+
+MAINTAINER = brice.fromentin
+DESCRIPTION = Free. Cross-platform. Open source. A developer platform for building all apps. This package installs the dotnet runtime and the asp.net.core runtime.
+CHANGELOG = "dotnet-runtime v9.0.0 release."
+
+STARTABLE = no
+DISPLAY_NAME = .NET 9 runtime
+
+HOMEPAGE = https://dotnet.microsoft.com/
+LICENSE  = MIT
+
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+# we do not only need the updated libstdc++ library, we also need to
+# adjust the library search path for dotnet to use this version.
+DEPENDS += cross/libstdc++
+POST_STRIP_TARGET = dotnet_patch_target
+endif
```

Comment:
If you are using `REQUIRED_MIN_DSM = 7.0` as in line 9, this is not required. The `OPTIONAL_DEPENDS = cross/libstdc++` on line 6 will also be redundant.

Reply:
@brice-fromentin to be clear, this section of code checks if the version of DSM is less than 7.0. If your required minimum DSM version is greater than or equal to 7.0 then this entire section should be removed from your code.